torch/csrc/jit/operator_upgraders/upgraders.h

```diff
@@ -0,0 +1,76 @@
+#pragma once
+#include <string>
+#include <unordered_map>
+
+namespace torch {
+namespace jit {
+
+static std::unordered_map<std::string, std::string> upgraders_graph(
```

Comment:
Is it temporary to have IR text in the file? When will we be able to compile python upgraders on the fly during `torch.jit.load`?

Reply:
https://github.com/pytorch/pytorch/pull/68773 This is the stack of PRs that will make sure the upgraders are generated on the fly. This stack will be landed after that.